raw/index.rst

```diff
@@ -0,0 +1,128 @@
+loklak
+======
+
+'Lok Lak' is a very tasty Cambodian stir-fry meat dish with a lot of fresh 
+black pepper. I hope not to scare vegetarians with this name, currently I am 
+one as well. And, loklak is also the name of this project that the goal of 
+collecting and indexing social media message data in a peer to peer network 
+to enable users to analyze big data.
+
+loklak server
+-------------
+
+The loklak server is a server application which is able to collect messages 
+from various sources, including twitter. The server contains a search index 
+and a peer-to-peer index sharing interface. All messages are stored in an 
+elasticsearch index. An automatic deployment from the development branch at 
+GitHub is available for tests here https://loklak-server-dev.herokuapp.com
+
+Code: https://github.com/loklak/loklak_server
+
+Project Site: http://api.loklak.org
+
+Documentation: http://dev.loklak.org/server
```

Comment:
I was a bit skeptical about it as these links don't actually exist right now. Maybe we can make these changes later when we have the documentation of these sub-projects in the repository.